src/button/README.md

```diff
@@ -2,7 +2,43 @@
 
 The `vscode-button` is a web component implementation of a [button element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button). The `vscode-button` also supports several visual appearances––primary, secondary, and icon.
 
-## Attributes
+![Button hero](/docs/assets/images/button-hero.png)
+
+## Usage
+
+### Types
+
+| Type      | Example                                                                                               | Usage                                                                              |
+| --------- | ----------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| Primary   | <img src="../../docs/assets/images/button-types-primary.png" alt="Primary button" width="100px"/>     | Emphasizes the highest priority action in a view.                                  |
+| Secondary | <img src="../../docs/assets/images/button-types-secondary.png" alt="Secondary button" width="100px"/> | Used for additional actions in a view that already features a primary action.      |
+| Icon      | <img src="../../docs/assets/images/button-types-icon.png" alt="Icon button" width="50px"/>            | A space-efficient style that renders a single icon to represent a specific action. |
+
+### Best Practices
+
+| ❌ Don't                                                           | ✅ Do                                                                              |
+| ------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| ![Multiple primary buttons](/docs/assets/images/button-dont-1.png) | ![One primary and multiple secondary buttons](/docs/assets/images/button-do-1.png) |
+| Don't use multiple primary buttons in close proximity.             | Provide a single primary button with one or more secondary actions                 |
+
+| ❌ Don't                                                                | ✅ Do                                                               |
+| ----------------------------------------------------------------------- | ------------------------------------------------------------------- |
+| ![Buttons with incorrect casing](/docs/assets/images/button-dont-2.png) | ![Buttons with correct casing](/docs/assets/images/button-do-2.png) |
+| Don't use all caps or lowercase text.                                   | Use sentence case for all button text.                              |
```

Comment:
I would lean towards abundantly clear/accessible language by changing: 

"Don't use all caps or lowercase text. " to "Don't use fully capitalized or lowercase text. "

Or something along those lines